server/eventlog_test.go

```diff
@@ -633,18 +669,71 @@ func generateNonCosCelEvent(hashAlgoList []crypto.Hash) (cel.Record, error) {
 	return randRecord, nil
 }
 
+func TestParseLinuxKernelState(t *testing.T) {
+	logs := []struct {
+		eventLog
+		name            string
+		expectedCmdline string
+	}{
+		{COS85AmdSev, "COS85AmdSev", test.Cos85AmdSevCmdline},
+		{COS93AmdSev, "COS93AmdSev", test.Cos93AmdSevCmdline},
+		{COS101AmdSev, "COS101AmdSev", test.Cos101AmdSevCmdline},
+	}
+	for _, log := range logs {
+		for _, bank := range log.Banks {
+			hashName := pb.HashAlgo_name[int32(bank.Hash)]
+			subtestName := fmt.Sprintf("%s-%s", log.name, hashName)
+			t.Run(subtestName, func(t *testing.T) {
+				msState, err := parsePCClientEventLog(log.RawLog, bank, GRUB)
+				if err != nil {
+					t.Errorf("failed to parse and replay log: %v", err)
+				}
+
+				if msState.LinuxKernel == nil || len(msState.LinuxKernel.CommandLine) == 0 {
+					t.Errorf("expected %s to have a LinuxKernelState", log.name)
+				}
+
+				if msState.LinuxKernel.CommandLine != log.expectedCmdline {
+					got := len(msState.LinuxKernel.CommandLine)
+					want := len(log.expectedCmdline)
+					if got != want {
+						t.Log(got)
+						t.Log(want)
```

Comment:
This only log a number, could make the message more specific/detail

Reply:
These are left over debugging lines; thanks for catching. Removed